flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/history/HistoryServerArchiveFetcher.java

```diff
@@ -79,9 +110,15 @@
 	private final JobArchiveFetcherTask fetcherTask;
 	private final long refreshIntervalMillis;
 
-	HistoryServerArchiveFetcher(long refreshIntervalMillis, List<HistoryServer.RefreshLocation> refreshDirs, File webDir, CountDownLatch numArchivedJobs) {
+	HistoryServerArchiveFetcher(
+		long refreshIntervalMillis,
+		List<HistoryServer.RefreshLocation> refreshDirs,
+		File webDir,
+		Consumer<JobArchiveEvent> jobArchiveEventListener,
+		boolean processArchiveDeletion
```

Comment:
```suggestion
		boolean cleanupExpiredArchives
```

or whatever the name of this is in the `HistoryServer`.

Reply:
Renamed. Thanks